docs/basics/primitives.md

```diff
@@ -39,7 +204,7 @@ that helps educate Phantom and simultaneously Cassandra of what your type `means
 "Deriving" is very simple, if we have a primitive for a type `T` that already exists, then we can define a new primitive
 for any type `X` we want, provided there is a bijection from `T` to `X`, or in simple terms a way to convert a `T
  to an `X` and an `X` back to a `T` without losing any details in the process.
- 
+
 The signature of `derive` looks like this. We use a context bound on `Source` to tell the compiler
```

Comment:
Just:
We use a context bound on `Source` to tell the compiler
we expect type `Source` to have a pre-defined primitive, and the two parameters are the conversion
functions. The output is a `Primitive[Target]`, as anticipated.